src/components/layout/AppHeader.vue

```diff
@@ -1,6 +1,7 @@
 <template>
 	<div class="navigation">
-		<a class="navigation-left" :href="`/?${ campaignParams }`" aria-hidden="true">
+		<a class="navigation-left" :href="`/?${ campaignParams }`" tabindex="-1">
```

Comment:
Can you remove `tabindex="-1"` from this? We'll let it be tabbable.

Apart from that it's all good.